packages/block-editor/src/store/reducer.js

```diff
@@ -689,7 +689,8 @@ const withReplaceInnerBlocks = ( reducer ) => ( state, action ) => {
 			index: 0,
 		} );
 
-		// We need to re-attach the block order of the controlled inner blocks.
+		// We need to re-attach the controlled inner blocks to the blocks tree and
+		// preserve their block order.
```

Comment:
Can we remove the line break between the sentences? Reads a bit awkward. Also there are two periods after entity to fix :)